modules/checklist/steps/setup-header.php

```diff
@@ -11,7 +11,7 @@
 }
 
 class Setup_Header extends Step_Base {
-	const STEP_ID = 'setup_header';
+	const STEP_ID = 'setup_header_promo';
```

Comment:
I do not like this, because we've implemented class-autoload as `new {namespace}\{step-id}` , but it's half of the story, we also know about this class in the PRO repository.
main idea behind the changes was to make them in such a way, that PRO repository doesn't have to be changed for all to work...